### PR TITLE
Placed intermediate button on the toolbar

### PR DESCRIPTION
**Pull request by Domily11**

### CODE REVIEW

File: leaf-ui/css/style.css [hunk 1]
```diff
@@ -456,6 +456,8 @@ select > .select-placeholder {
   display: inline;
 }
```
Comment (leshghi) [inline leaf-ui/css/style.css]: Remove this file from commit

File: leaf-ui/js/initializeElements.js [hunk 2]
```diff
@@ -78,8 +78,6 @@ var model = new Model();
 var analysisRequest = new AnalysisRequest();        //TODO: make not global!
 var analysisResult = new AnalysisResult();
 
-
```
Comment (leshghi) [inline leaf-ui/js/initializeElements.js]: Remove this file from commit

File: leaf-ui/css/analysis_style.css [hunk 3]
```diff
@@ -375,7 +375,7 @@ select::-ms-expand {
     cursor: pointer;
   }
```
Comment (leshghi) [inline leaf-ui/css/analysis_style.css]: Remove this file from commit

File: leaf-ui/rappid-extensions/AnalysisInspector.js [hunk 4]
```diff
@@ -14,6 +14,8 @@ var AnalysisInspector = Backbone.View.extend({
 		'<div class="inspector-views">',
 		'<h2 style="text-align:center; width:100%;margin-top:6px;margin-bottom:0px">Analysis</h2>',
 		'<hr>',
+		//'<button id="btn-view-intermediate" class="analysis-btns inspector-btn sub-label green-btn">Intermediate Values</button>',
```
Comment (leshghi) [inline leaf-ui/rappid-extensions/AnalysisInspector.js]: Can delete this

File: leaf-ui/rappid-extensions/AnalysisInspector.js [hunk 5]
```diff
@@ -25,16 +27,14 @@ var AnalysisInspector = Backbone.View.extend({
...
-		'<hr>',
 		'<button id="btn-single-path" class="analysis-btns inspector-btn sub-label green-btn">Simulate Single Path</button>',
 		'<button id="btn-all-next-state" class="analysis-btns inspector-btn sub-label ice-btn">Explore Possible Next States</button>',
 		'<hr>',
 		'</div>',
 		'</script>'].join(''),		
 	
 	events: {
-		'click #btn-view-intermediate': 'openIntermediateValuesTable',
+		//'click #btn-view-intermediate': 'openIntermediateValuesTable',
```
Comment (leshghi) [inline leaf-ui/rappid-extensions/AnalysisInspector.js]: Can delete this line

File: leaf-ui/rappid-extensions/AnalysisInspector.js [hunk 6]
```diff
@@ -92,13 +92,14 @@ var AnalysisInspector = Backbone.View.extend({
 	/**
 	 * Creates, attaches, and renders view for the IVT
 	 */
+	/**
```
Comment (leshghi) [inline leaf-ui/rappid-extensions/AnalysisInspector.js]: Can delete this function

File: leaf-ui/rappid-extensions/AssignmentsTable.js [hunk 7]
```diff
@@ -126,8 +126,8 @@ var AssignmentsTable = Backbone.View.extend({
     loadRelativeIntentions: function(){
         for (let constraint in this.model.get('constraints')){
             var relIntentionRow = new RelativeIntentionView({model: constraint, new: false});
-        $('#rel-intention-assigments').append(relIntentionRow.el);
-        relIntentionRow.render();
+            $('#rel-intention-assigments').append(relIntentionRow.el);
```
Comment (leshghi) [inline leaf-ui/rappid-extensions/AssignmentsTable.js]: Remove this file from commit

File: leaf-ui/rappid-extensions/BackboneModelsOther.js [hunk 8]
```diff
@@ -27,10 +27,10 @@
     },
 
     defaults: {
-        linkDestID: null,
-        linkType:  'AND',
+        linkType:  'and',
```
Comment (leshghi) [inline leaf-ui/rappid-extensions/BackboneModelsOther.js]: Remove this file from commit

File: leaf-ui/rappid-extensions/LinkInspector.js [hunk 9]
```diff
@@ -1,16 +1,20 @@
 // Class for the Link properties tab that appears when link settings are clicked.
 
 var LinkInspector = Backbone.View.extend({
+    model: joint.dia.CellLink,
```
Comment (leshghi) [inline leaf-ui/rappid-extensions/LinkInspector.js]: Remove this file from commit

File: leaf-ui/rappid-extensions/joint.extensions.js [hunk 10]
```diff
@@ -191,25 +191,43 @@ joint.shapes.basic.Resource = joint.shapes.basic.Intention.extend({
     }
 });
 
-joint.dia.cellLink = joint.dia.Link.extend({
-    
+joint.dia.CellLink = joint.dia.Link.extend({
```
Comment (leshghi) [inline leaf-ui/rappid-extensions/joint.extensions.js]: Remove this file from commit

**Review by leshghi — CHANGES_REQUESTED** [review]
This looks great to me! Awesome job - thank you. I think you may need to pull from backbone-refactor to get some files/changes out of your pull request. Let me know if you need help getting those out. Also - would you be able to put a thin line between the Assignments List button and the Intermediate Values button? No worries if not but I think it could help separate the two visually.

File: leaf-ui/css/style.css (hunk 1 above)
Comment (Domily11) [reply leaf-ui/css/style.css]: Pls help!!

**Review by leshghi — APPROVED** [review]
Looks great to me. Outstanding issue where you cannot close the IVT but that is fixed in the ae-ivt-backbone branch where we are working on the IVT file so not related to your PR at all. You're all set - thanks

**Pull request merged**